Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM 056154071827.dkr.ecr.us-east-1.amazonaws.com/base-image-ruby-version-arg:2.3
+FROM 056154071827.dkr.ecr.us-east-1.amazonaws.com/base-image-ruby-version-arg:2.5
```

Comment:
@Omicron7 not sure about the Docker image (can not access the registry), is this enough?

Reply:
Yeah, that works.